arm-recoveryservices/2015-01-01/swagger/recoveryservices.json

```diff
@@ -0,0 +1,285 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-01-01",
+    "title": "RecoveryServicesBackupVaultClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json",
+    "application/octet-stream"
```

Comment:
why do you need "application/octet-stream". I don't see any operation in this spec needing this. 
Does the service actually support "text/json"? If not then please remove it.